src/generic/create-or-rerun-course/CreateOrRerunCourseForm.jsx

```diff
@@ -192,7 +227,7 @@ const CreateOrRerunCourseForm = ({
             key={field.label}
           >
             <Form.Label>{field.label}</Form.Label>
-            {!field.isDropdown ? (
+            {field.name !== 'org' ? (
```

Comment:
maybe create const with this string?

Reply:
I think it's gonna be a little redundant.